src/SFA.DAS.EmployerAccounts/Extensions/AuthenticationServiceExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.Authentication;
+using System.Security.Claims;
+
+namespace SFA.DAS.EmployerAccounts.Extensions
+{
+    public static class AuthenticationServiceExtensions
+    {
+        public static bool IsSupportUser(this IAuthenticationService authenticationService)
+        {
+            return authenticationService.HasClaim(ClaimsIdentity.DefaultRoleClaimType, "Tier2User");
```

Comment:
Change `Tier2User` to make use of the constants declared across the projects? or ok to leave as is?